scripts/bring_assisted_service.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+export SERVICE_REPO=${SERVICE_REPO:-}
+export OPENSHIFT_CI=${OPENSHIFT_CI:-flase}
+export REPO_NAME=${REPO_NAME:-assisted-service}
+export JOB_TYPE=${JOB_TYPE:-}
+export PULL_BASE_REF=${PULL_BASE_REF:-master}
+
+
+
+if ! [[ -d "assisted-service" ]]; then
+  echo "Can't find assisted-service source locally, cloning ${SERVICE_REPO}"
+  git clone "${SERVICE_REPO}"
+fi
+
+service_active_branch=$(cd assisted-service/ && git rev-parse --abbrev-ref HEAD)
+pr_branch_name=assisted-service-pr-${PULL_NUMBER}
+
+if [[ "${OPENSHIFT_CI}" = "true" && "${REPO_NAME}" = "assisted-service" && "${JOB_TYPE}" = "presubmit" ]]; then
+  if [[ "${service_active_branch}" = "${pr_branch_name}" ]]; then
+    echo "Nothing to update. ${REPO_NAME} is already updated, branch: ${pr_branch_name}"
+    exit 0
+  fi
+
+  echo "Running in assisted-service pull request"
+	cd assisted-service && \
+	git fetch -v origin "pull/${PULL_NUMBER}/head:${pr_branch_name}" && \
+	git checkout "${pr_branch_name}" && \
```

Comment:
some weird indentation here

Reply:
and instead of ``&&`` we can make each line fail inherently with ``set``

Reply:
Done